### PR TITLE
Courses Completed Onboarding Screen

### PR DESCRIPTION
**Pull request by imanmoreira**
Implements the next onboarding screen that allows students to fill out which courses they have completed thus far. This will auto-populate the schedule into fall and spring semesters and display how many courses they have completed so far.

### CODE REVIEW

**Comment by yasz24** [comment]
[Completed courses/pre-populating schedule, on-boarding](https://trello.com/c/aRD9FZir/25-completed-courses-pre-populating-schedule-on-boarding)

File: frontend/src/Onboarding/CompletedCoursesScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+        course.subject,
+        String(course.classId)
+      );
+      if (scheduleCourse) {
+        this.setState({
+          selectedCourses: [...this.state.selectedCourses, scheduleCourse],
+        });
+      }
+    } else {
+      console.log("filtered");
```
Comment (ArunJ98) [inline frontend/src/Onboarding/CompletedCoursesScreen.tsx:168]: Do we need this console log?

File: frontend/src/Onboarding/CompletedCoursesScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+  onExpand(requirementGroup: string, change: Boolean) {
+    this.state.expandedSections.set(requirementGroup, change);
+    this.setState({
+      expandedSections: this.state.expandedSections,
+    });
+  }
+
+  // renders the link for show more or show less for the specific requirementGroup, depending on the boolean
+  renderShowLink(requirementGroup: string, more: boolean) {
+    let variable = more ? "more" : "less";
```
Comment (ArunJ98) [inline frontend/src/Onboarding/CompletedCoursesScreen.tsx:189]: minor stylistic thing: Can we rename variable to something more descriptive?

File: frontend/src/Onboarding/CompletedCoursesScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+    );
+  }
+
+  // Renders one course/courseset (if it contains labs/recitiations, and seperated)
+  renderCourse(courses: IRequiredCourse[]) {
+    let allCourse = courses.map(course => course.subject + course.classId);
+    return (
+      <CourseWrapper key={allCourse[0]}>
+        <Checkbox
+          // checked={this.state.selectedCourses.some((s) => s.subject == courses[0].subject &&
```
Comment (ArunJ98) [inline frontend/src/Onboarding/CompletedCoursesScreen.tsx:210]: Do we need this commented out code, or can we take it out?

**Pull request merged**